wolfcrypt/src/asn.c

```diff
@@ -15973,8 +15973,8 @@ static int ConfirmSignature(SignatureCtx* sigCtx,
             #if !defined(NO_DSA) && !defined(HAVE_SELFTEST)
                 case DSAk:
                 {
-                    ret = wc_DsaVerify(sigCtx->digest, sigCtx->sigCpy,
-                            sigCtx->key.dsa, &sigCtx->verify);
+                    ret = wc_DsaVerify_ex(sigCtx->digest, WC_SHA_DIGEST_SIZE,
```

Comment:
Something is not right with `ConfirmSignature` I had to change this to `WC_SHA_DIGEST_SIZE` despite the test csr hashing the signature with sha256 and setting the digestSz to 32. I'm not sure if this is a part of CSR logic that I don't understand but it doesn't pass when I try to use the full 32 bytes